shaderc-sys/Cargo.toml

```diff
@@ -15,9 +15,9 @@ links = "shaderc"
 name = "shaderc_sys"
 
 [features]
-check-passthrough = []
-dont-use-deprecated = []
 build-from-source = []
+check-inverted-no-defaults = []
+inverted-no-defaults = []
```

Comment:
How do you think about `shaderc-rs-use-sys-libs` and `use-sys-libs` here then? Given the tight relationship between shaderc-rs and shaderc-sys, it is reasonable to have an option dedicated to shaderc-rs here. (I'm just trying to make the feature more self-documenting. :) )

Reply:
This crazy inversion thing is only necessary because of how `--no-default-features` operates on a crate's config.  I would support a rename to `shaderc` specific if you keep the backwards compatibility.  If you axe the backwards compatibility, these features go away.  `shaderc-rs-inverted-no-defaults` etc would be more appropriate.  The features have no control over reading specific directories, only the old behavior of reading the `../out/` directory.

If you decide to axe no defaults, it's as simple as removing these two features in both crates and axing the block in build.rs that reads them, then removing the documents.

The reason I implemented the `--no-default-features` backwards compatibility is because you believed people were using it (possibly in CI?).  It's wise to maintain at least one release of deprecation in any case.

Reply:
SG. This will be removed soon, so I won't bother that much anymore. :)